frontend/app/src/components/SmallButton/index.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
```

Comment:
Small button could be same as with Button, but with different props

Reply:
Yes tried it but ran into problems so to go forward I just made a new component :D